updates to credentials for demo apps

oraNod opened the pull request:
suggested revisions for the section that lists demo credentials for portal and mobile apps

oraNod committed:
updates to credentials for demo apps

suggested revisions for the section that lists demo credentials for portal and mobile apps

wtrocki commented:
I'm not quite sure what was the issue here. 
Can you explain what was the problem?

oraNod commented:
I changed a couple of things here:
- removed '.Credentials for the Mobile and Portal Applications' because it seems like it inserts a redundant heading or something. I just think this can reasonably be dropped so the text is more concise.
- put the credentials for the portal app before the credentials for the mobile app. this order follows the order for starting the apps.
- moved the note below the table.
- updated the link text for the json file because it's more specific than 'github repository'
- the text about the sample content is moved to the beginning of the section on running the applications.
- I don't see the reason for pointing to 'xref:building-wfm-process'. IMO this would get the user off topic. whereas the flow for the "Getting Started" section should lead users to the next section for "Learning about Workflows"
- other minor changes for consistent wording

wtrocki commented:
> I don't see the reason for pointing to 'xref:building-wfm-process'. IMO this would get the user off topic. whereas the flow for the "Getting Started" section should lead users to the next section for "Learning about Workflows"

It was pointed out that after starting applications users do not know what next. Getting started experience should require minimal time. I think it's worth to have this link to align with our video and have fast getting started experience.

wtrocki commented on docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc:
```diff
@@ -78,28 +78,23 @@ npm run start
-For a complete list of available users, see the link:https://github.com/feedhenry-raincatcher/raincatcher-server/blob/master/src/modules/passport-auth/users.json[gitub repository]
+For all available users, see: https://github.com/feedhenry-raincatcher/raincatcher-server/blob/master/src/modules/passport-auth/users.json[users.json]
 
-NOTE: The Mobile and Portal Applications contain sample Steps, Workflows and Workorders.
```
IMHO that is useful info. Can we revert that?

oraNod committed:
updates with review feedback

making some changes to address review feedback

oraNod commented:
added a new commit to address the changes from review. thanks!
